### ISSUE TITLE
Update indigent-defense-stats code to reflect updates in azure-indigent-defense code

### ISSUE THREAD

**Issue by nicolassaw** [open-austin/indigent-defense-stats#62]
Update the code from this repo's code to reflect the code of [this repo](https://github.com/open-austin/azure-indigent-defense/issues).

Focus on these functions:
- scraper
- parser
- cleaner
- other helpful data analysis, logging, tools, or features

Worry less about these functions:
- interacting with PowerApps or Azure products

**Comment by nicolassaw** [comment]
I compared the versions of the different parts of the code (scraper, parser, additional tools, any additional cleanup functions, etc.) and found many folders and pieces of code were related to the Azure/LogicApps infrastructure.

Example: Scraper: First part of the scraper is initializing the blob container and receiving parameters from a single request sent via JSON (we aren't doing this with the VM). Most everything else looks the same besides the additions and changes to accommodate the LogicApps infrastructure that doesn't need to be copied over.

However, I also found an update in 2023 to the parser that added handling for "also known as". This was added to the current parser. Very helpful.

Otherwise, closing this ticket out unless @normaljosh has some other ideas of things I might have missed.
